### DISCUSSION TITLE
How do I convert hex-colors to RGBA of Modelviewer

### DISCUSSION THREAD

**Discussion by Aztec91** [google/model-viewer#2290] [Q&A]
Hey there just a quick question for understanding.

let material2 = modelViewer.model.materials[2];
material2.pbrMetallicRoughness.setBaseColorFactor([1, 0, 0, 1]);

this code changes the 3rd Material of my model to red. Thats nice.
But I would love to set it to a specific hex-color. Like for example: #ff6600;

I see modelviewer workds with RGBA but the number only reach from 0-1 not from 0-255 as you would exspect.
So how can I properly convert Hex-Colors to the RGBA from Modelviewer?

Thanks for your help

Aztec

**Comment by elalish** [comment]
Divide by 255 😁
